styles/styles.css

```diff
@@ -498,7 +503,7 @@ main .why-us .default-content-wrapper p {
 }
 
 .job-description-page p:nth-child(4):hover,
-.job-description-page p:nth-child(8):hover {
+.job-description-page p:nth-child(7):hover {
```

Comment:
you can use semantics (like making text italic/strong) so as to avoid the numbering in p elements